### PR TITLE
269 - Fix text overflow

### LINKED ISSUE #269: Fix text overflow
Screen Shot 2024-04-29 at 7 09 36 PM

### PR DESCRIPTION
**Pull request by ghostman2311**
Closes #269

### CODE REVIEW

**Comment by buckyroberts** [comment]
Not fixed.

Screen Shot 2024-05-02 at 7 43 31 PM

File: src/components/Post/Comments/Comment/Styles.ts
Comment (MAbdurrehman1) [inline src/components/Post/Comments/Comment/Styles.ts:1]: Looks like this change has been made Only in Comments Styling, that's why the fix isn't working on post content

Comment (MAbdurrehman1) [reply src/components/Post/Comments/Comment/Styles.ts:1]: Ideally, I think there should be a common styling for this which can be reused for both, comments and post's content

**Review by MAbdurrehman1 — APPROVED** [review]
👍

**Comment by MAbdurrehman1** [comment]
image

**Pull request merged**